src/components/network-modification-tree.js

```diff
@@ -64,12 +67,34 @@ const NetworkModificationTree = ({
     const classes = useStyles();
 
     const selectedNode = useSelector((state) => state.selectedTreeNode);
+    const workingNode = useSelector((state) => state.workingTreeNode);
 
     const treeModel = useSelector(
         (state) => state.networkModificationTreeModel
     );
 
     const [isMoving, setIsMoving] = useState(false);
+    const [isMinimapOpen, setIsMinimapOpen] = useState(false);
+
+    const nodeColor = (node) => {
+        if (node.type === 'ROOT') {
+            return 'rgba(0, 0, 0, 0.0)';
+        } else {
+            if (node.id === workingNode?.id) {
+                return '#4287f5';
+            }
+            switch (node.data.buildStatus) {
+                case 'BUILT':
+                    return '#70d136';
+                case 'BUILT_INVALID':
+                    return '#9196a1';
+                case 'NOT_BUILT':
+                    return '#9196a1';
+                default:
+                    return '#9196a1';
```

Comment:
Why not using const INVALID_NODE_BANNER_COLOR = '#9196a1'; from the other file ?